Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.py

```diff
@@ -0,0 +1,395 @@
+        """
+        Get a list of the Risk Detection objects and their properties.
+
+        Args:
+            risk_state (str): Risk State to retrieve.
+            risk_level (str): Specify to get only results with the same Risk Level.
+            limit (int): Limit of results to retrieve.
+            skip_token (int): Skip token.
+
+        return:
+            Response (dict): API response from AzureRiskyUsers.
+        """
+        params = remove_empty_elements({'$top': limit,
+                                        '$skiptoken': skip_token,
+                                        '$filter': build_query_filter(risk_state, risk_level)})
+
+        return self.ms_client.http_request(method='GET',
+                                           url_suffix="/identityProtection/riskDetections",
+                                           params=params)
+
+    def risk_detection_get(self, id: str) -> dict:
+        """
+        Read the properties and relationships of a riskDetection object.
+
+        Args:
+            id (str): ID of risk detection to retrieve.
+
+        Return:
+            Response (dict): API response from AzureRiskyUsers.
+        """
+        return self.ms_client.http_request(method='GET',
+                                           url_suffix=f'/identityProtection/riskDetections/{id}')
+
+
+def build_query_filter(risk_state: Optional[str], risk_level: Optional[str]) -> Optional[str]:
+    """
+    Build query filter for API call, in order to get filtered results.
+
+    Args:
+        risk_state (str): Wanted risk state for filter.
+        risk_level (str): Wanted risk level for filter.
+
+    Returns:
+        str: Query filter string for API call.
+    """
+    if risk_state and risk_level:
+        return f"riskState eq '{risk_state}' and riskLevel eq '{risk_level}'"
+    elif risk_state:
+        return f"riskState eq '{risk_state}'"
+    elif risk_level:
+        return f"riskLevel eq '{risk_level}'"
+    else:
+        return None
+
+
+def create_event_or_incident_output(item: Dict,
+                                    table_headers: List[str]) -> Dict[str, Optional[Any]]:
+    """
+    Create the complete output dictionary for events or incidents.
+
+    Args:
+        item (dict): A source dictionary from the API response.
+        table_headers (list(str)): The table headers to be used when creating initial data.
+
+    Returns:
+        object_data (dict(str)): The output dictionary.
+    """
+    return remove_empty_elements({field: item.get(field) for field in table_headers})
+
+
+def risky_users_list_command(client: Client, args: Dict[str, str]) -> CommandResults:
+    """
+    List all risky users.
+    Args:
+        client (Client): Azure Risky Users API client.
+        args (dict): Arguments for API call.
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    limit = int(args['limit'])
+    page = int(args['page'])
```

Comment:
1. limit and page aren't mandatory args, how can we access them this way?
2. Also, what if it isn't an int? an ugly error would be raised. You can use the arg to int function from CSP

Reply:
In my previous integration CR , I was told to do it this way for arguments with default values.
But I will change it.

Reply:
Done